root/static/styles/entity.less

```diff
@@ -23,23 +22,28 @@ div.tabs {
             a {
                 display: block;
                 padding: 0.1em 0.5em;
-                margin-left: 4px;
+                margin-right: 2px;
```

Comment:
I never liked the left margin really, and I think with this design it looks even weirder. Switching to right.